jenkins/kola/packet.sh

```diff
@@ -27,6 +27,29 @@ else
   CPIO_URL="http://${BASE_PATH}/flatcar_production_pxe_image.cpio.gz"
 fi
 
+if [[ "${BOARD}" == "amd64-usr" ]]; then
+  for INSTANCE in c3.small.x86 c3.medium.x86 m3.large.x86 s3.xlarge.x86 n2.xlarge.x86; do
+    timeout --signal=SIGQUIT "${timeout}" bin/kola run \
+    --basename="${NAME}" \
+    --board="${BOARD}" \
+    --channel="${GROUP}" \
+    --gce-json-key="${UPLOAD_CREDS}" \
+    --packet-api-key="${PACKET_API_KEY}" \
+    --packet-facility="${PACKET_REGION}" \
+    --packet-image-url="${IMAGE_URL}" \
+    --packet-installer-image-kernel-url="${KERNEL_URL}" \
+    --packet-installer-image-cpio-url="${CPIO_URL}" \
+    --packet-project="${PACKET_PROJECT}" \
+    --packet-storage-url="${UPLOAD_ROOT}/mantle/packet" \
+    --packet-plan="${INSTANCE}" \
+    --parallel="${PARALLEL_TESTS}" \
+    --platform=packet \
+    --tapfile="${JOB_NAME##*/}_validate_${INSTANCE}.tap" \
+    --torcx-manifest=torcx_manifest.json \
+    cl.internet || true
+  done
+fi
```

Comment:
```suggestion
    cl.internet || true &
  done
fi

wait
```
We can kick these off in parallel if we don't care about the result (`|| true`) anyway.

Reply:
Yes, I just think that it makes the stdout logs worthless if they are mangled

Reply:
I pushed a change that starts all in parallel but collects the output to print it in one go. It can still become mangled but less.